jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/TypeUtil.java

```diff
@@ -174,18 +175,31 @@ public static <T> List<T> asList(T[] a)
     }
 
     /**
-     * <p>Returns a new list with the elements of the specified list in reverse order.</p>
-     * <p>The specified list is not modified, differently from {@link Collections#reverse(List)}.</p>
-     *
-     * @param list the list whose elements are to be reversed
-     * @return a new list with the elements in reverse order
+     * <p>Returns a {@link ListIterator} positioned at the last item in a list</p>
+     * <p>The method is safe for concurrent modifications iff the list iterator itself is safe.</p>
```

Comment:
I'd remove this line, "the method is safe" does not mean much, and the properties of the iterator are those of the list passed in, so the caller knows -- this method just tries to position the iterator at the end.

Reply:
```suggestion
```